packages/unmock-jsdom/src/persistence/local-storage-persistence.ts

```diff
@@ -0,0 +1,51 @@
+import * as yml from "js-yaml";
+import * as path from "path";
+import { IPersistableData, IPersistence } from "unmock";
+
+// Uses directory structure for keys in window local storage
+const UNMOCK_KEY = ".unmock";
+const TOKEN_KEY = ".token";
+const TOKEN_FULL_KEY = path.join(UNMOCK_KEY, TOKEN_KEY);
+const SAVE_PATH = path.join(UNMOCK_KEY, "save");
+const UNMOCK_FILE = "unmock.yml";
+
+export default class LocalStoragePersistence implements IPersistence {
```

Comment:
Might be a good time to rename this to something more descriptive. IMO `LocalStorage` hints at file system related storage. Maybe `BrowserStorage` could be a good alternative?

Reply:
done!